verticapy/tests/vModel/test_decision_tree_classifier.py

```diff
@@ -223,6 +223,21 @@ def test_to_sklearn(self, model):
 
         # 'predict_proba'
 
+    def test_to_python(self, model, titanic_vd):
+        model_test = DecisionTreeClassifier("rfc_python_test", cursor=model.cursor)
+        model_test.drop()
+        model_test.fit(titanic_vd, ["age", "fare", "sex"], "embarked")
+        model_test.cursor.execute(
+            "SELECT PREDICT_RF_CLASSIFIER(* USING PARAMETERS model_name = 'rfc_python_test', match_by_pos=True) FROM (SELECT 30.0 AS age, 45.0 AS fare, 'male' AS sex) x"
```

Comment:
I think you can simplify the query to:

"SELECT PREDICT_RF_CLASSIFIER(30.0, 45.0, 'male' USING PARAMETERS model_name = 'rfc_python_test', match_by_pos=True)"

Reply:
When the dataset is unbalanced, the number of bins is not enough to get a precise AUC

Reply:
Done;